tests/test_zephyr_interface_no_ci.py

```diff
@@ -0,0 +1,220 @@
+            "testCase",
+            "environment",
+            "jiraProjectVersion",
+            "testExecutionStatus",
+            "actualEndDate",
+            "estimatedTime",
+            "executionTime",
+            "executedById",
+            "assignedToId",
+            "comment",
+            "automated",
+            "testCycle",
+            "customFields",
+            "links",
+        ]
+
+        test_execution_id = "BLOCK-E192"
+        test_execution = await self.zapi.get_test_execution(test_execution_id)
+        self.assertListEqual(list(test_execution.keys()), payload_expected_keys)
+        self.assertEqual(test_execution["key"], test_execution_id)
+
+    async def test_extract_test_case_from_test_execution(self):
+
+        expected_test_case_name = "BLOCK-T21"
+        expected_test_case_version = "1"
+
+        test_execution_id = "BLOCK-E192"
+        test_execution = await self.zapi.get_test_execution(test_execution_id)
+        test_case_name, test_case_version = (
+            self.zapi.extract_test_case_from_test_execution(test_execution)
+        )
+
+        self.assertEqual(test_case_name, expected_test_case_name)
+        self.assertEqual(test_case_version, expected_test_case_version)
+
+    async def test_get_test_executions(self):
+
+        payload_expected_keys = [
+            "next",
+            "startAt",
+            "maxResults",
+            "total",
+            "isLast",
+            "values",
+        ]
+
+        values_expected_keys = [
+            "id",
+            "key",
+            "project",
+            "testCase",
+            "environment",
+            "jiraProjectVersion",
+            "testExecutionStatus",
+            "actualEndDate",
+            "estimatedTime",
+            "executionTime",
+            "executedById",
+            "assignedToId",
+            "comment",
+            "automated",
+            "testCycle",
+            "customFields",
+            "links",
+        ]
+
+        test_cycle_key = "BLOCK-R21"
+        test_executions = await self.zapi.get_test_executions(test_cycle_key)
+        self.assertListEqual(list(test_executions.keys()), payload_expected_keys)
+        self.assertListEqual(
+            list(test_executions["values"][0].keys()), values_expected_keys
+        )
+
+    async def test_parse_environment_from_id(self):
+
+        environment_id = 6824992  # Daytime environment id for '1. Daytime'
+        environment = await self.zapi.parse_environment_from_id(environment_id)
+        self.assertEqual(environment, "1. Daytime")
+
+    async def test_parse_project_from_id(self):
+
+        project_id = 350001  # BLOCK project id
+        project = await self.zapi.parse_project_from_id(project_id)
+        self.assertEqual(project, "BLOCK")
+
+    async def test_parse_status_from_id(self):
+
+        status_id = 3940035  # Pass status id
+        status = await self.zapi.parse_status_from_id(status_id)
+        self.assertEqual(status, "Pass")
```

Comment:
I suggest to also add tests for:
- `parse_test_case_from_id`
- `parse_test_cycle_from_id`

Reply:
Another suggestion that could be too much, but could be good for better maintenance: move all constants such as `environment_id`, `project_id`, `status_id` and even others as e.g. `test_cycle_key` (in other tests) to a config file (e.g. `tests/config.py` or `tests/constants.py`).

Reply:
When writing these tests, I found out that there is no API interface that I can use to write `parse_test_case_from_id` . So I am removing any reference to it.

Reply:
When writing these tests, I found out that there is no API interface that I can use to write `parse_test_case_from_id` . So I am removing any reference to it.

Reply:
Regarding your second comment, I created global variables at the beginning of `tests/test_zephyr_interface_no_ci.py`. Not as pretty as you suggested, but it is a quick-and-dirty solution.

Reply:
having global variables at the beginning sounds great too! thanks for looking into this.